portfolio/src/main/webapp/script.js

```diff
@@ -27,23 +30,19 @@ function addRandomFunFact() {
   funfactContainer.innerText = funfact;
 }
 
-// TODO: Use following functions in step 5 to display comment
-
+// TODO: Use following functions in step 5 to display comment.
 function getComment() {
   fetch('/data').then(response => response.json()).then((entries) => {
-
-  const quoteListElement = document.getElementById('entry-container');
-  quoteListElement.appendChild(
-      createQuoteElement('Name: ' + entries.name));
-  quoteListElement.appendChild(
-      createQuoteElement('Email: ' + entries.email));
-  quoteListElement.appendChild(
-      createQuoteElement('Comment: ' + entries.comment));
+    console.log(entries);
+    const quoteListElement = document.getElementById('entry-container');
+    entries.forEach((entry) => {
```

Comment:
rename quoteListElement to commentListElement.

Reply:
Done